### PR TITLE
rexec: add page

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/rexec.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# rexec
+
+> Execute a command on a remote host.
+> Use with caution, as `rexec` transmits data in plain text. Consider secure alternatives like `ssh` for encrypted communication.
```
Comment (kbdharun) [inline pages/linux/rexec.md:4]:
```suggestion
> Note: Use `rexec` with caution, as it transmits data in plain text. Consider secure alternatives like `ssh` for encrypted communication.
```
How about this for the description?

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/linux/rexec.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+`rexec -username={{username}} -h={{remote_host}} ps aux`
+
+- Redirect `stdin` from `/dev/null` on a remote [h]ost:
+
+`rexec --no-err -h={{remote_host}} ls -l`
+
+- Specify the remote [P]ort on a remote [h]ost:
+
+`rexec -P={{1234}} -h={{remote_host}} ls -l`
```
Comment (gutjuri) [inline pages/linux/rexec.md:21]:
```suggestion
- Execute a command on a remote [h]ost:

`rexec -h={{remote_host}} {{ls -l}}`

- Specify the remote [u]sername on a remote [h]ost:

`rexec -username={{username}} -h={{remote_host}} {{ps aux}}`

- Redirect `stdin` from `/dev/null` on a remote [h]ost:

`rexec --no-err -h={{remote_host}} {{ls -l}}`

- Specify the remote [P]ort on a remote [h]ost:

`rexec -P={{1234}} -h={{remote_host}} {{ls -l}}`
```

**Review by gutjuri — CHANGES_REQUESTED** [review]
Thanks! One suggestion, marking commands to be executed on remote as placeholders.

**Review by gutjuri — APPROVED** [review]

**Pull request merged**